addons/binding/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/BindingConstants.java

```diff
@@ -15,10 +15,10 @@
 import com.google.common.collect.ImmutableSet;
 
 /**
- * This class defines common constants, which are used
- * across the whole binding.
+ * This class defines common constants, which are used across the whole binding.
  *
  * @author Robert Bausdorf
+ * @author Christoph Weitkamp
```

Comment:
Please mention what you added/changed, maybe also add something like ' - initial contribution' to his name?